python/runtime/db.py

```diff
@@ -304,10 +304,7 @@ def reader():
 
 
 def pai_maxcompute_db_generator(table,
-                                feature_column_names,
-                                label_column_name,
-                                feature_metas,
-                                fetch_size=128,
+                                label_column_name=None,
```

Comment:
This change seems is not related to this PR?

Reply:
Yes. Removed.